spec/helpers/override_helper_spec.rb

```diff
@@ -1,6 +1,32 @@
 require 'rails_helper'
 
 RSpec.describe OverrideHelper do
+  let(:allocation_one) {
```

Comment:
more idiomatic to call 'build' on a factory so that the test only sets the fields it cares about?

Reply:
RSpec complains with: 
`build` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks). 
 As I use them across two different describe blocks I pulled them out to avoid code duplication

Reply:
Scrap that - done!